docs/examples/tree-v2/filter.vue

```diff
@@ -15,7 +15,7 @@
 <script lang="ts" setup>
 import { ref } from 'vue'
 import { ElTreeV2 } from 'element-plus'
-import type { TreeNode } from 'element-plus/es/components/tree-v2/src/types'
+import type { TreeNode } from 'element-plus/es/components/tree-v2'
```

Comment:
I don't think we should modify this now because `TreeNode` a is not exported from `element-plus/es/components/tree-v2`